src/app/manage/twitch/events-overview/events-overview.component.scss

```diff
@@ -0,0 +1,30 @@
+@import '../../../../utils/mixins';
+
+.eventsList {
+  @include gridLayout;
```

Comment:
Since we have that on 3 pages, I think I'll just extract it as a component (or two, depending on the items) , then there would be only one style to fix and not 3 (and probably counting) 

that way also each "addNew" button will be only one component (one style)